src/opnsense/mvc/app/views/OPNsense/Diagnostics/arp.volt

```diff
@@ -61,7 +61,17 @@ POSSIBILITY OF SUCH DAMAGE.
                     }
             );
         }
+        
+        
+        function flushARP() {
+            ajaxGet(url = "/api/diagnostics/interface/flushArp",
+                sendData = {}, callback = function (data, status) {
+                    $("#refresh").click();
+                });
+        }
 
+        $("#flush").click(flushARP);
```

Comment:
@peppelinux probably better to wrap all in the .click() action and add a dialog to warn the user, an arp flush can have side affects

Reply:
I'm looking for a bootstrap class template or whatever builtin in OPN UI

Reply:
you can use bootstrapdialog, for example https://github.com/opnsense/core/blob/c14000892b603a70c931aca44db01021e49d12d8/src/www/interfaces_groups.php#L71

Reply:
Ops, I was late to read it. I'll use this next time, very very thanks

Reply:
can you please simplify this code? a simple bootstrap dialog containing the ajaxCall(....) should be more then enough.

Reply:
Ok, If it's important I will soon as possible

Reply:
Done, you was right, it's much better then before